policy/policyd_test.go

```diff
@@ -133,14 +134,15 @@ func Test_policy_StartPolicyUpdater(t *testing.T) {
 			return test{
 				name: "Start updator success",
```

Comment:
update test case name?
`"Start updator success"` - > `"Start success"`

Reply:
LGTM, @kevindiu  can you fix it?

Reply:
I think the meaning is the same, for test case we do not need to care about that detail, as everyone can understand it. Anyway I will update it this time.